devops/git/workflow.md

```diff
@@ -9,7 +9,8 @@ after the initial commit, with the correct protection rules in place.
 
 * `master` should be the working development branch [UNSTABLE]
 * `feature/*` branches created from master for each feature
-* `hotfix/*` branches should only be created off of `staging` or `production` and merged accordingly.
+* `fix/*` branches are for non-urgent fixes, and follow the same flow as feature branches.
+* `hotfix/*` branches should only be created off of `staging` or `production` and merged accordingly for urgent fixes only.
```

Comment:
It might be something to mention that a `hotfix` branch should be merged into `master` after it's been merged into either `production` or `staging` to make sure that the fix is in the development branch.